usaspending_api/api_docs/api_documentation/advanced_award_search/spending_by_geography.md

```diff
@@ -0,0 +1,305 @@
+## [Spending by Geography](#spending-by-geography)
+**Route:** `/api/v2/search/spending_by_geography/`
+
+**Method:** `POST`
+
+This route takes award filters, and returns spending by state code, county code, or congressional district code.
+
+### Request
+scope: Scope defines what location is used for filtering the aggregated_amount.
+
+geo_layer: Defines which geographical level should be returned in the request. Options include: "state", "county", "district"
+
+geo_layer_filter: Defines a filter for a specific geographic area correlating to the geo_layer. It is a list of strings that are the unique identifiers for the geographic location.
+
+- When `geo_layer` is `"state"` then  `the geo_layer_filters` should be the county `shape_code` ex: `["MN", "WA", "DC"]`. The state is the `state_code`
+- When `geo_layer` is `"county"` then  the `the geo_layer_filters` should be the county `geo_id` ex: `["51041", "51117", "51179"]`. The county `shape_code` is a concatenation of the `state_FIPS_code` + the `county_code`
```

Comment:
Change to:

The county `shape_code` is a concatenation of the state FIPS code and the county FIPS code.

Reply:
DOne,  missed this one